firmware/projects/TMS/platforms/windows/CMakeLists.txt

```diff
@@ -1,2 +1,2 @@
-target_sources(bindings PRIVATE bindings.cc)
+target_sources(bindings PRIVATE bindings.cc os.cc)
```

Comment:
we don't have os for windows. There shouldn't be an os.cc file under platforms/windows